source/installation/raspberrypi.markdown

```diff
@@ -57,6 +57,15 @@ If Raspberry Pi Imager is not supported by your platform, you can [download the
 2. Plug in an Ethernet cable and make sure the Raspberry Pi is connected to the same network as your computer and is connected to the internet.
 3. Connect the power supply to start up the device.
 
+### Alternately if you don't have Ethernet connection
```

Comment:
We strongly discourage using Wi-Fi, additionally, the instruction below are making it an advanced use case. I am in doubt if we should add this at all for this reason.

Even if we add it, it should be wrapped in a details block to hide it from the general instructions.

Reply:
Oh I see. I do understand that Ethernet is much better and stable. 

Looking forward to improving the docs in other ways though.